deploy/clowdapp.yaml

```diff
@@ -225,13 +225,61 @@ parameters:
     value: "1"
   - name: PULP_WEB_REPLICAS
     description: Number of pulp web replicas
-    value: "1"
+    value: "0"
```

Comment:
Makes sense to leave a comment here why we're setting `0` here?

Reply:
yeah ... i'll add one